src/components/SelectedOption/__tests__/SelectedOption.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SelectedOption } from '../SelectedOption';
+
+describe('<SelectedOption>', () => {
+    const onDeleteMock = jest.fn();
+
+    let view: RenderResult;
+
+    afterEach(() => {
```

Comment:
I don't think we need this anymore. It was an older version of jest that required manual reset.